docs/modules/ROOT/pages/api/erc20.adoc

```diff
@@ -141,52 +110,102 @@ Emitted when the allowance of a `spender` for an `owner` is set.
 `value` is the new allowance.
 
 [.contract]
-[[ERC20]]
-=== `++ERC20++` link:https://github.com/OpenZeppelin/cairo-contracts/blob/cairo-2/src/token/erc20/erc20.cairo[{github-icon},role=heading-link]
+[[IERC20Metadata]]
+=== `++IERC20Metadata++` link:https://github.com/OpenZeppelin/cairo-contracts/blob/cairo-2/src/token/erc20/interface.cairo#L6-L19[{github-icon},role=heading-link]
 
 [.hljs-theme-dark]
 ```javascript
-use openzeppelin::token::erc20::ERC20;
+use openzeppelin::token::erc20::interface::IERC20Metadata;
 ```
 
-Implementation of the <<IERC20,IERC20>> interface.
+IERC20Metadata interface as defined in {eip20}.
```

Comment:
`IERC20Metadata` is not really defined in EIP20, the standard defines a single ERC20 interface with three optional methods